frontend/packages/jobs/src/prompts/generateSchemaMeta/generateSchemaMeta.ts

```diff
@@ -60,33 +177,109 @@ export const generateSchemaMeta = async (
   runId: string,
   schemaFiles: string,
 ) => {
-  const model = new ChatOpenAI({
+  const evaluationModel = new ChatOpenAI({
+    model: 'o3-mini',
```

Comment:
I used cline's model selection guide as a reference and adopted the reasoning model for the plan.

https://docs.cline.bot/getting-started/model-selection-guide#planning